scripts/tank_cmd.py

```diff
@@ -1230,7 +1254,148 @@ def run_engine_cmd(log, pipeline_config_root, context_items, command, using_cwd,
         return tank_command.run_action(log, tk, ctx, command, args)
 
 
+def _extract_args(cmd_line, *args):
```

Comment:
Why not just `args` or `arg_list`? Seems `*args` adds complexity for no good reason. What if we wanted to add a future parameter?